### DISCUSSION TITLE
How optuna suggests parameters?

### DISCUSSION THREAD

**Discussion by kkckk1110** [optuna/optuna#5290] [Q&A]
### Don't use GitHub Issues to ask support questions.

Hello! I found optuna really helpful in optimizing parameters. But I have two questions:
1. What does a trial mean? Does it simply mean one set of parameters?
2. How does optuna suggest parameters? Does it totally select a random number between the given intervals without any selections? Or, does it have some search directions?
I am looking forward to your responses.

**Comment by nzw0301** [comment]
1. For basic usage and terms of optuna, please check https://optuna.readthedocs.io/en/stable/tutorial/index.html
2. The sampler list https://optuna.readthedocs.io/en/stable/reference/samplers/index.html. Note that the default sampler depends on the number of objectives which is specified by `study`.
